### ISSUE TITLE
`Remote-Connect: Try a Sample...` Fails on `docker volume create` command

### ISSUE THREAD

**Issue by itanex** [microsoft/vscode-remote-release#4386]
- VSCode Version: Version: 1.52.1 (user setup)
- Local OS Version: Windows 10.0.19042
- Remote OS Version: WSL2 Debian GNU/Linux 10 (Buster)
- Remote Extension/Connection Type: Docker

Steps to Reproduce:

1. Open Command Palette in VSCode
2. Execute Remote-Connect: Try a Sample
3. Select ANY of the provided images (This occurs for all presented images)

~~Does this issue occur when you try this locally?: Yes/No~~
~~Does this issue occur when you try this locally and all extensions are disabled?: Yes/No~~

Behavior:

1. VSCode opens an remote folder
2. Attempts to pull and build the image
3. Docker build error occurs when docker tries to create the volume

Terminal Log:

```
[5 ms] Remote-Containers 0.155.1 in VS Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523).
[21 ms] Start: Resolving Remote
[24 ms] Start: Check Docker is running
[24 ms] Start: Run: docker version --format {{.Server.APIVersion}}
[175 ms] Server API version: 1.41
[176 ms] Start: Run: docker volume ls -q
[464 ms] Start: Run: docker volume create --label vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-node --label vsch.local.repository.unique=false vsc-remote-sample
[528 ms] Command failed: docker volume create --label vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-node --label vsch.local.repository.unique=false vsc-remote-sample
```

**Comment by bamurtaugh** [comment]
@Chuxel Does this sound like the issue we discussed a few weeks ago that involved running Docker Desktop in WSL and not being able to find try-* in WSL?

**Comment by itanex** [comment]
@bamurtaugh I saw that ticket that was closed but I was not able to use its guidance to get things working for me. Plus it was failing a different step in the docker build.

**Comment by bamurtaugh** [comment]
Thanks for the additional info @itanex. And really glad you enjoyed the session, thanks so much for joining 😄

**Comment by itanex** [comment]
@bamurtaugh I stumbled upon my actual issue and the solution. 

I was attempting to build out my first `docker-compose.yml` and when building I was getting the following error.

> ERROR: The platform targeted with the current context is not supported.

After investigating what this meant. I ran the context list command. It appears that my context was not switched back after removing an image (using the docker extension in VSC) that was created in another tutorial. Docker for windows application does not display contexts so, novices like me, working with Docker may not be aware of this pitfall.

I switched the context back to the default context, manually removed that tutorials context and tested this feature out once more. It now works as expected.

Action plan:

In the [Remote development in Containers](https://code.visualstudio.com/docs/remote/containers-tutorial) tutorial, add a note that instructs the user to set a specific context/create a context to ensure this error does not occur.

Change proposed in PR [vscode-docs#4279](https://github.com/microsoft/vscode-docs/pull/4279)
